### PR TITLE
Bump lighthouse versions to 2.0.5

### PR DESCRIPTION
**Pull request by pault84**
Signed-off-by: Paul <paul@portworx.com>

**What this PR does / why we need it**:
Bump lighthouse version and dependend packages to 2.0.5

### CODE REVIEW

File: charts/portworx/values.yaml [hunk 1]
```diff
@@ -27,9 +27,9 @@ customRegistryURL:
 registrySecret:
 
 lighthouse: true
-lighthouseVersion: 2.0.4
-lighthouseSyncVersion: 0.3
-lighthouseStorkConnectorVersion: 0.1
+lighthouseVersion: 2.0.5
+lighthouseSyncVersion: 2.0.5
```
Comment (harsh-px) [inline charts/portworx/values.yaml]: did we sync all these versions to 2.0.5?

Comment (harsh-px) [reply charts/portworx/values.yaml]: https://install.portworx.com/2.1 is using

portworx/px-lighthouse:2.0.4
portworx/lh-config-sync:0.4
portworx/lh-stork-connector:0.2

Helm should be at the same versions

**Comment by rodrigc** [comment]
@pault84 Your change doesn't fully work.  I'm getting this:

```
  Warning  Failed     4m10s (x3 over 4m54s)   kubelet, ip-192-168-83-222.us-west-2.compute.internal  Failed to pull image "portworx/lh-stork-connector:2.0.5": rpc error: code = Unknown desc = Error response from daemon: manifest for portworx/lh-stork-connector:2.0.5 not found
```

File: charts/portworx/values.yaml (hunk 1 above)
Comment (pault84) [reply charts/portworx/values.yaml]: We need 2.0.5. for a fix regarding disks. there is no difference between the other pieces minus that. we're just pulling versions in line. If we need to update install.portworx.com then so be it.

**Comment by pault84** [comment]
> @pault84 Your change doesn't fully work. I'm getting this:
> 
> ```
>   Warning  Failed     4m10s (x3 over 4m54s)   kubelet, ip-192-168-83-222.us-west-2.compute.internal  Failed to pull image "portworx/lh-stork-connector:2.0.5": rpc error: code = Unknown desc = Error response from daemon: manifest for portworx/lh-stork-connector:2.0.5 not found
> ```

That means we haven't pushed that version i'll ask shailvi to do that now.

File: charts/portworx/values.yaml (hunk 1 above)
Comment (pault84) [reply charts/portworx/values.yaml]: We started syncing them for the operator.
Next step is just having 1 version in this values file and using that for all the images.

**Comment by rodrigc** [comment]
It works now:

```
kubectl -n kube-system describe pod px-lighthouse-56c6d89b75-6647f

...
 Type    Reason     Age   From                                                   Message
  ----    ------     ----  ----                                                   -------
  Normal  Scheduled  98s   default-scheduler                                      Successfully assigned kube-system/px-lighthouse-56c6d89b75-6647f to ip-192-168-83-222.us-west-2.compute.internal
  Normal  Pulling    96s   kubelet, ip-192-168-83-222.us-west-2.compute.internal  pulling image "portworx/lh-config-sync:2.0.5"
  Normal  Created    94s   kubelet, ip-192-168-83-222.us-west-2.compute.internal  Created container
  Normal  Started    94s   kubelet, ip-192-168-83-222.us-west-2.compute.internal  Started container
  Normal  Pulled     94s   kubelet, ip-192-168-83-222.us-west-2.compute.internal  Successfully pulled image "portworx/lh-config-sync:2.0.5"
  Normal  Pulling    93s   kubelet, ip-192-168-83-222.us-west-2.compute.internal  pulling image "portworx/px-lighthouse:2.0.5"
  Normal  Pulled     92s   kubelet, ip-192-168-83-222.us-west-2.compute.internal  Successfully pulled image "portworx/px-lighthouse:2.0.5"
  Normal  Created    92s   kubelet, ip-192-168-83-222.us-west-2.compute.internal  Created container
  Normal  Pulling    91s   kubelet, ip-192-168-83-222.us-west-2.compute.internal  pulling image "portworx/lh-config-sync:2.0.5"
  Normal  Started    91s   kubelet, ip-192-168-83-222.us-west-2.compute.internal  Started container
  Normal  Pulled     90s   kubelet, ip-192-168-83-222.us-west-2.compute.internal  Successfully pulled image "portworx/lh-config-sync:2.0.5"
  Normal  Created    90s   kubelet, ip-192-168-83-222.us-west-2.compute.internal  Created container
  Normal  Started    90s   kubelet, ip-192-168-83-222.us-west-2.compute.internal  Started container
  Normal  Pulling    90s   kubelet, ip-192-168-83-222.us-west-2.compute.internal  pulling image "portworx/lh-stork-connector:2.0.5"
  Normal  Pulled     89s   kubelet, ip-192-168-83-222.us-west-2.compute.internal  Successfully pulled image "portworx/lh-stork-connector:2.0.5"
  Normal  Created    89s   kubelet, ip-192-168-83-222.us-west-2.compute.internal  Created container
  Normal  Started    89s   kubelet, ip-192-168-83-222.us-west-2.compute.internal  Started container
```


```
kubectl -n kube-system get pods



NAME                                       READY   STATUS      RESTARTS   AGE
aws-node-56nvk                             1/1     Running     0          115m
aws-node-gh967                             1/1     Running     0          115m
aws-node-wsxlf                             1/1     Running     0          115m
coredns-67b9c4d455-s5p4v                   1/1     Running     0          121m
coredns-67b9c4d455-zp824                   1/1     Running     0          121m
kube-proxy-snwcr                           1/1     Running     0          115m
kube-proxy-z597w                           1/1     Running     0          115m
kube-proxy-zqhr5                           1/1     Running     0          115m
portworx-67tnn                             1/1     Running     0          3m9s
portworx-8xm2s                             1/1     Running     0          3m9s
portworx-j9t6d                             1/1     Running     0          3m9s
portworx-pvc-controller-7866ddbf67-cx8s7   1/1     Running     0          3m9s
portworx-pvc-controller-7866ddbf67-wkb6x   1/1     Running     0          3m9s
portworx-pvc-controller-7866ddbf67-wsh4p   1/1     Running     0          3m9s
px-hook-etcd-preinstall-5rgd4              0/1     Completed   0          3m20s
px-hook-postdelete-unlabelnode-99kcf       0/1     Completed   0          4m30s
px-hook-predelete-nodelabel-r26dl          0/1     Completed   0          4m46s
px-lighthouse-56c6d89b75-6647f             3/3     Running     0          3m9s
stork-64db84d848-85rjm                     1/1     Running     0          3m9s
stork-64db84d848-jjltn                     1/1     Running     0          3m9s
stork-64db84d848-nn7r5                     1/1     Running     0          3m9s
stork-scheduler-74d5b5cd97-7fj5r           1/1     Running     0          3m9s
stork-scheduler-74d5b5cd97-88t6n           1/1     Running     0          3m9s
stork-scheduler-74d5b5cd97-skdlp           1/1     Running     0          3m9s
tiller-deploy-6db5b44cf-ttlk7              1/1     Running     0          108m
```

File: charts/portworx/values.yaml (hunk 1 above)
Comment (harsh-px) [reply charts/portworx/values.yaml]: Okay. 

Please send the px-installer PR too in that case. They need to be in sync.

**Comment by pault84** [comment]
Will send the pr to 2.1 tonight

**Pull request merged**